rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2023-09-28"
-components = ["rustfmt-preview", "rustc-dev", "rust-src", "miri"]
+channel = "stable"
+components = ["rustfmt"]
```

Comment:
We still need `rust-src` for documentation in vscode

Reply:
Fixed